src/test/java/gov/healthit/chpl/aqa/pageObjects/BasePage.java

```diff
@@ -59,4 +61,14 @@ public static WebElement headerNav(final WebDriver driver) {
     public static WebElement pageTitle(final WebDriver driver, final String pageTitle) {
         return driver.findElement(By.xpath("//h1[contains(text(),'" + pageTitle + "')]"));
     }
+
+    public static List<WebElement> getRowElements(final WebElement row) {
```

Comment:
If you were to describe this method in English, it would be something like:

> Given a table
> Return all of the rows in that table

Right? If that's the case, then the "input" WebElement shouldn't be named "row", but instead be named "table", and the output element should definitely not be "roWebElements" (also note the missing w in "row") but instead just be "rows".

Similar comments for "getColElements"